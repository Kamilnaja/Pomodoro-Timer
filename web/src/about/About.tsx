import { Container } from 'react-bootstrap';
import './about.scss';

export const About = () => {
  return (
    <Container className="about">
      <h2>About</h2>
      <h3>What is this page made for?</h3>
      <p>TomatoToDone is an pomodoro application, that helps you achieve more in live and work.</p>
      <p>Due to pomodoro technique, work for 25 minutes and then take 5 minutes break.</p>
      <p>Repeat!</p>
    </Container>
  );
};
