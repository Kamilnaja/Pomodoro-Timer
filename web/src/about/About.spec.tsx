import { shallow, ShallowWrapper } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import { About } from './About';

describe('About', () => {
  let wrapper: ShallowWrapper;

  beforeEach(() => {
    wrapper = shallow(<About />);
  });

  it('should match snapshot', () => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
