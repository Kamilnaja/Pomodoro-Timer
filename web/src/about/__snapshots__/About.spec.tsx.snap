// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should match snapshot 1`] = `
<Container
  fluid={false}
>
  <h2>
    About
  </h2>
  <h3>
    What is this page made for?
  </h3>
  <p>
    TomatoToDone is an pomodoro application, that helps you achieve more in live and work.
  </p>
  <p>
    Due to pomodoro technique, work for 25 minutes and then take 5 minutes break.
  </p>
  <p>
    Repeat!
  </p>
</Container>
`;
